### DISCUSSION TITLE
4.9 version documentation link ???

### DISCUSSION THREAD

**Discussion by SanshooSenshi** [Palakis/obs-websocket#1105] [General]
can't get to find the documentation for 4.9 websocket version 
the link below doesn't seems to work anymore ...
https://github.com/obsproject/obs-websocket/blob/4.x-current/docs/generated/protocol.md
where I could find it plz

**Comment by willyk0** [comment]
Here you are:

https://github.com/obsproject/obs-websocket/blob/4.x-compat/docs/generated/protocol.md

Version 5.1 is still very very very broken by design, long live to 4.9!
